cfgov/unprocessed/js/modules/util/dom-events.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+/**
+ * Parses the event passed by the bind shortcut to create the event listener
+ * @param   {string}   evt      The type of event to watch for
+ * @param   {HTMLNode} elem     The element to watch for the passed event
+ * @param   {Function} callback The function to trigger on the event
+ */
+function _parseEvent( elem, evt, callback ) {
+  evt.split( /\s+/ ).forEach(
+    function( event ) {
+      elem.addEventListener( event, callback );
+    }
+  );
+}
+
+/**
+ * Shortcut for binding event listeners to elements
+ * @param  {HTMLNode} elem    The element to attach the event listener to
+ * @param  {Object}   options The options for the event listener
```

Comment:
I've updated the comment, it's an object containing each of the events to attach.